pom.xml

```diff
@@ -16,7 +16,7 @@
     <properties>
         <!-- Adapt the jenkins.tools.bom artifact ID in the dependencyManagement section
              if the Jenkins core version is changed -->
-        <jenkins.version>2.319.1</jenkins.version>
+        <jenkins.version>2.332.3</jenkins.version>
```

Comment:
If the Jenkins version is changed, the Jenkins bom artifact should be adapted accordingly as well.

Reply:
Yes, exactly. I tried to reflect this in the comment above. Maybe we should rewrite it accordingly to improve the intention behind it.

Reply:
@romanisb  OK now?

Reply:
Looks good.